deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -463,7 +464,7 @@ class ChatViewController: UITableViewController {
         nc.removeObserver(self, name: UIApplication.didBecomeActiveNotification, object: nil)
         nc.removeObserver(self, name: UIApplication.willResignActiveNotification, object: nil)
         audioController.stopAnyOngoingPlaying()
-
+        
```

Comment:
whitespace error, for whatever reason, that prevents rebasing, so better fix that :)